content/faq/_index.md

```diff
@@ -27,8 +27,27 @@ If you encounter a problem while using Packit-as-a-service, please open an
 In case of any other questions, feel free to contact us:
 **user-cont-team@redhat.com**.
 
-### How can I download RPM spec file if it is not part of upstream repository?
+### Why do I have to maintain .packit.yaml and a spec file upstream?
+[We are working](https://github.com/packit-service/packit/issues/574) on simplifying the `.packit.yaml` so it's as small as possible.
+We will also handle all potentialy backward incompatible changes of `.packit.yaml`.
```

Comment:
```suggestion
We will also handle all potentially backwards incompatible changes of `.packit.yaml`.
```
my gut tells me it's usually spelled like this

Reply:
ehm, no :) https://www.quickanddirtytips.com/education/grammar/backward-versus-backwards

Reply:
Now I'm even more confused.